### PR TITLE
bugfix: look for "unzip" and "7zip", not ".*.exe"

### PR DESCRIPTION
**Pull request by CeresBarros**

### CODE REVIEW

File: R/copy.R [hunk 1]
```diff
@@ -9,9 +9,10 @@
 file.move <- function(from, to, overwrite = FALSE) {
   stopifnot(file.exists(from))
   res <- suppressWarnings(file.rename(from = from, to = to))
-  if (.isFALSE(res)) {
+
+  if (!isTRUE(all(res))) {
     res2 <- file.copy(from = from, to = to, overwrite = overwrite)
-    if (isTRUE(res2)) {
+    if (!isTRUE(all(res))) {
```
Comment (eliotmcintire) [inline R/copy.R:15]: This needs no negation

Comment (CeresBarros) [reply R/copy.R:15]: oops!

File: R/copy.R [hunk 2]
```diff
@@ -9,9 +9,10 @@
 file.move <- function(from, to, overwrite = FALSE) {
   stopifnot(file.exists(from))
   res <- suppressWarnings(file.rename(from = from, to = to))
-  if (.isFALSE(res)) {
+
+  if (!isTRUE(all(res))) {
     res2 <- file.copy(from = from, to = to, overwrite = overwrite)
-    if (isTRUE(res2)) {
+    if (isTRUE(all(res))) {
```
Comment (eliotmcintire) [inline R/copy.R:15]: Needs to be `res2` not `res`

Comment (CeresBarros) [reply R/copy.R:15]: thius is what happens when you copy paste! sorry!

**Review by eliotmcintire — APPROVED** [review]

**Pull request merged**